src/icons/CheckBoxSelected.tsx

```diff
@@ -0,0 +1,31 @@
+import { createSvgIcon } from '@mui/material'
+
+const CheckBoxSelected = createSvgIcon(
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    stroke="currentColor"
+  >
+    <rect
+      x="5.625"
+      y="5.625"
+      width="12.75"
+      height="12.75"
+      rx="1.375"
+      stroke="#currentColor"
```

Comment:
```suggestion
      stroke="currentColor"
```
오타가 있네요 :( 아래도 같은 에러가 있으니 확인 부탁드려요